### ISSUE TITLE
Migrate linter to conda-verify

### ISSUE THREAD

**Issue by jakirkham** [conda-forge/conda-smithy#512]
One of the points at today's meeting regards the future of `conda-smithy`'s linter. In particular, migration of its content out of `conda-smithy` into some separate package. ( https://github.com/conda-forge/conda-smithy/issues/386 ) ( https://github.com/conda-forge/conda-smithy/issues/61#issuecomment-185398366 )

@msarahan raised a good point that [`conda-verify`]( https://github.com/conda/conda-verify ) is well-positioned to take on that role. IMHO this sounds like a great idea. Though it would be really good to hear from others on this change. Am reaching out to people who have been involved with the behavior of our linter and otherwise rely on its use in conda-forge. Please share your thoughts.

cc @conda-forge/core @conda-forge/staged-recipes @goanpeca @janschulz @kynan @CJ-Wright @dopplershift @krischer @minrk

**Comment by goanpeca** [comment]
>@msarahan raised a good point that conda-verify is well-positioned to take on that role.

I think it makes perfect sense to have it as a separate thing. Also ti would be a good opportunity to generalize it and make it more extendable.

**Comment by CJ-Wright** [comment]
I'd be on board with this.

**Comment by msarahan** [comment]
For anyone who missed the meeting: the intent behind creating conda-verify was to make a (community-oriented) place that everyone could add validators for recipes and for packages in a central location.  Since everyone has their own standards, everyone would be free to select different validators, or perhaps pass different arguments to particular validators.  I also wanted validation to not be tied to a particular ecosystem.  For example, conda-forge's linter bot should not necessarily need conda - just python - and conda-verify should ideally be a tool that the linter bot can just use.

I have not been successful in getting conda-verify (which was a fork of anaconda-verify) to replace anaconda-verify.  Conda-build has integrated conda-verify for quite a while now.  I doubt that it is perfect, but I hope that it will be a useful tool.

**Comment by johanneskoester** [comment]
@daler @bgruening maybe we should consider conda verify as well.

**Comment by bgruening** [comment]
Definitely, we discussed this with @daler and we can easily extend our linter. I think we needed to agree on - all rules or only a few.

**Comment by isuruf** [comment]
We now have very specific lints like checking that the github handle of the maintainer is valid. I guess we need a generic way to do this using `conda-verify`. https://github.com/conda/conda-verify/issues/10
